src/tokens/transformation/tailwind/convertCssToJs.js

```diff
@@ -6,12 +6,15 @@ const postcss = require('postcss')
 const cssFiles = ['variables.css']
```

Comment:
Not for Tailwind, since the different layers build to distinct folders.

```
tailwind
├── browser
│  ├── index.js
│  ├── package.json
│  ├── plugins
│  ├── tokens.js
│  ├── typography.js
│  └── variables.css
├── package.json
├── plugins
│  └── components
└── universal
   ├── index.js
   ├── package.json
   ├── plugins
   ├── tokens.js
   ├── typography.js
   └── variables.css
```